bigquery/biglake/bigquery_create_unpartitioned_biglake_table/main.tf

```diff
@@ -0,0 +1,116 @@
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+/**
+* This Terraform code sample creates a BigLake table in
+* Google Cloud Storage with an unpartitioned schema.
+* For more information, see
+* https://cloud.google.com/bigquery/docs/create-cloud-storage-table-biglake
+* and
+* https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/bigquery_table
+*/
+
+# [START bigquery_create_biglake_unpartitioned_table]
+# This creates a bucket in the US region named "my-bucket" with a pseudorandom suffix.
+resource "random_id" "bucket_name_suffix" {
+  byte_length = 8
+}
+resource "google_storage_bucket" "default" {
+  name                        = "my-bucket-${random_id.bucket_name_suffix.hex}"
+  location                    = "US"
+  force_destroy               = true
+  uniform_bucket_level_access = true
+}
+
+# This queries the provider for project information.
+data "google_project" "project" {}
+
+# This creates a connection in the US region named "my-connection".
+# This connection is used to access the bucket.
+resource "google_bigquery_connection" "default" {
+  connection_id = "my-connection"
+  location      = "US"
+  cloud_resource {}
+}
+
+# This grants the previous connection IAM role access to the bucket.
+resource "google_project_iam_member" "default" {
+  role    = "roles/storage.objectViewer"
+  project = data.google_project.project.id
+  member  = "serviceAccount:${google_bigquery_connection.default.cloud_resource[0].service_account_id}"
+}
+
+# This makes the script wait for seven minutes before proceeding.
+# This lets IAM permissions propagate.
+resource "time_sleep" "wait_7_min" {
+  depends_on      = [google_project_iam_member.default]
+  create_duration = "7m"
+}
+
+# This defines a Google BigQuery dataset with
+# default expiration times for partitions and tables, a
+# description, a location, and a maximum time travel.
+resource "google_bigquery_dataset" "default" {
+  dataset_id                      = "my_dataset"
+  default_partition_expiration_ms = 2592000000  # 30 days
+  default_table_expiration_ms     = 31536000000 # 365 days
+  description                     = "My dataset description"
+  location                        = "US"
+  max_time_travel_hours           = 96 # 4 days
+
+  # This defines a map of labels for the bucket resource,
+  # including the labels "billing_group" and "pii".
+  labels = {
+    billing_group = "accounting",
+    pii           = "sensitive"
+  }
+}
+
+
+# This creates a BigQuery table named "my_table" in the dataset "default".
+# The table has three columns, named "country", "product", and "price", which are of types STRING, STRING, and INT64 respectively.
+resource "google_bigquery_table" "default" {
+  depends_on = [time_sleep.wait_7_min]
+  dataset_id = google_bigquery_dataset.default.dataset_id
+  table_id   = "my_table"
+  schema     = <<EOF
```

Comment:
I see this was taken from https://cloud.google.com/bigquery/docs/create-cloud-storage-table-biglake#terraform, can we fix the indentation here (and there)?

Reply:
Just fixing it once here then importing this example into the Terraform tab of that doc page should work.

Reply:
done.

Reply:
i applied `jsonencode` to get rid of those confusing EOFs